samples/README.md

```diff
@@ -2,7 +2,7 @@
 
 Code samples for the documentation website reside in [core/.../test/.../samples/api](../core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api)
 (for samples that depend solely on `dataframe-core`),
-and [tests/.../samples/api](../tests/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api) (for samples can depend on other I/O modules)
+and [tests/.../samples/api](/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api) (for samples can depend on other I/O modules)
 and they are copied over to Markdown files in [docs/StardustDocs/topics](../docs/StardustDocs/topics)
 by [Korro](https://github.com/devcrocod/korro).
 
```

Comment:
Could you check any other readme's (like in :core) now also point here instead of /tests?